### ISSUE TITLE
Bug: Doesn't mask password keys when type is `json`

### ISSUE THREAD

**Issue by vyazadji** [fullstack-build/tslog#57]
**Describe the bug**
It doesn't mask password when type is `json`

**To Reproduce**
tslog is v2.10.0

When type is `pretty`, it masks `password` key
```
const log = new Logger()
log.info({ name: 'name1', password: 'secretPassword' })
```
output:
```
0|worker  | 2020-11-03 15:42:45.388.000
0|worker  |  INFO
0|worker  | [src/util.js:25]
0|worker  |
0|worker  | {
0|worker  |   name: 'name1',
0|worker  |   password: '[***]'
0|worker  | }
```
It works as expected and the password was masked. 

But when type is `json`
```
const log = new Logger({type: 'json'})
log.info({ name: 'name1', password: 'secretPassword' })
```
Output:
```
0|worker  | {"instanceName":"localhost","hostname":"localhost","date":"2020-11-03T15:50:11.733Z","logLevel":"info","logLevelId":3,"filePath":"src/util.js","fullFilePath":"/src/util.js","fileName":"util.js","lineNumber":25,"columnNumber":5,"isConstructor":false,"functionName":null,"typeName":"Object","methodName":null,"argumentsArray":["{ name: 'name1', password: 'secretPassword' }"]}
```
Password value wasn't masked in this case

**Expected behavior**
Password value should be masked for type `json`

**Node.js Version**
v10.23.0

**Comment by terehov** [comment]
This was actually done by design, where `type: "json"` gives you native access to the underlying object and the `pretty` option does everything to make it pretty. However, I see, that this behavior is not described properly in the documentation. 
In the last couple of days, I've been debating with myself whether this should be part of the library or something that the user would have to build on top of this "bare" access.

Eventually, I decided that the API would be misleading when it offers such configuration as masking without being active for one of the output types. 
So I agree with your issue and going to handle masking for `type: "json"` as well. In fact, I have already started implementing this 👍
